src/morphology.rs

```diff
@@ -361,12 +364,17 @@ pub struct Mask {
     /// for any optimisation/arithmetic purposes, it is garanteed that :
     ///
     /// - all the integer values will be strictly between -512 and 512
-    /// - all tuples will be sorted in lexicographic order
+    /// - all tuples will be sorted in reverse lexicographic order, line by line ((-1,-1),(0,-1),(1,-1),(-1,0),(0,0),...)
     /// - no tuple shall appear twice
-    elements: Vec<(i16, i16)>,
+    elements: Vec<Point<i16>>,
 }
 
 impl Mask {
+    fn new(elements: Vec<Point<i16>>) -> Self {
```

Comment:
Should we make this public to allow custom `Mask`s?

Reply:
No